pages/nft/iscn/_iscnId.vue

```diff
@@ -320,15 +332,10 @@ export default class NFTTestMintPage extends Vue {
       this.apiData = data
 
       if (this.hasOpener) {
-        try {
-          window.opener.postMessage(JSON.stringify({
-            action: 'NFT_MINT_DATA',
-            data,
-          }), this.redirectOrigin);
-        } catch (error) {
-          // eslint-disable-next-line no-console
-          console.error(error);
-        }
+        window.opener.postMessage(JSON.stringify({
+          action: 'NFT_MINT_DATA',
+          data,
+        }), this.redirectOrigin)
```

Comment:
Why remove the try catch??

Reply:
Will the try catch outside of this one do the same job?

Reply:
not really, I explicitly want to catch any error related to `window.opener.postMessage`, which `window.opener` is something we cannot control and might yield ignore-able error on postMessage